CONTRIBUTING.md

```diff
@@ -64,6 +64,33 @@ Administrative notes:
 - The [OWNERS file spec] is a useful resources in making changes.
 - Maintainers and admins must be added to the appropriate lists in both [Kustomize OWNERS_ALIASES] and [SIG-CLI Teams]. If this isn't done, the individual in question will lack either PR approval rights (Kustomize list) or the appropriate Github repository permissions (community list).
 
+## Project/Product Managers
+
+Kustomize will have opportunities to join in a project/product manager role. You can
+typically start working on this role as part of a kustomize training cohort, so please
+keep an eye out for that or reach out to the leads if you are interested in this type of
+work. Expectations for this role are:
```

Comment:
Non-blocking: Could the PM also be responsible for building or at least documenting the order for some sort of PR or issue-tackling queue? I'm frequently concerned that I missed an email or have been ignoring an old contribution.

Reply:
The goal if I understand you correctly is that they should be helping us easily discover if an old issue or PR has been waiting for us for a long time.

That sounds good to me, I'll add that.

Reply:
Looks great! Thank you.